packages/ember-htmlbars/tests/helpers/closure_component_test.js

```diff
@@ -309,4 +310,70 @@ if (isEnabled('ember-contextual-components')) {
     runAppend(component);
     equal(component.$().text(), 'Hodi Sergio', 'component is rendered');
   });
+
+  QUnit.test('raises an assertion when component path is null', function() {
+    let Mandarin = Component.extend();
+    registry.register(
+      'component:-mandarin',
+      Mandarin
+    );
+    registry.register(
+      'template:components/-mandarin',
+      compile(`ni hao`)
+    );
+    registry.register(
+      'template:components/-hindi',
+      compile(`Namaste`)
+    );
+
+    let template = compile(`{{component (component lookupComponent)}}`);
+    component = Component.extend({ container, template }).create();
+
+    expectAssertion(() => {
+      runAppend(component);
+    });
+  });
+
+  QUnit.test('raises an assertion when component path is null', function() {
```

Comment:
This test and the above are the same (even the same name), however this one asserts the message. Perhaps the previous test should be removed?